src/antidote_crdt_set_rw.erl

```diff
@@ -144,7 +145,9 @@ unique() ->
 
 -spec update(downstream_op(), set()) -> {ok, set()}.
   update(DownstreamOp, RWSet) ->
-    {ok, apply_downstreams(DownstreamOp, RWSet)}.
+    RWSet1 = apply_downstreams(DownstreamOp, RWSet),
+    RWSet2 = [Entry || {_, {AddTokens, RemoveTokens}} = Entry <- RWSet1, AddTokens =/= [] orelse RemoveTokens =/= []],
+    {ok, RWSet2}.
```

Comment:
When reset() is called, all seen tokens are reset and none are added. In this situation, the set would have an entry like {a, [], []}.